editor/js/Viewport.Controls.js

```diff
@@ -84,7 +86,13 @@ function ViewportControls( editor ) {
 		}
 
 		cameraSelect.setOptions( options );
-		cameraSelect.setValue( editor.viewportCamera.uuid );
+
+		const selectedCamera = ( editor.viewportCamera.uuid in options )
+			? editor.viewportCamera
+			: Object.values( editor.cameras )[ 0 ];
```

Comment:
We can use `editor.camera.uuid` here as well.

Reply:
Good.